### ISSUE TITLE
Suggestion for /findip

### ISSUE THREAD

**Issue by gomalley411** [TotalFreedom/TotalFreedomMod#437]
Maybe we should make it so that admins can see all IPs that have logged in as them (if they want)? Like, add in an argument that searches their UUID and gives them all the IPs that logged in as them so the admin will know (more or less) who posed as them? Just a thought.

**Comment by Wild1145** [comment]
Sounds like it links to #432

**Comment by looperXD** [comment]
I agree. Someone recently imposed as me and luckily the admins found out before any harm were done.
